pm_icecon/land_spillover.py

```diff
@@ -0,0 +1,125 @@
+"""Routines which implement land spillover corrections to sea ice concentration fields.
+
+land_spillover.py
+"""
+
+import os
+
+import numpy as np
+from scipy.signal import convolve2d
+
+
+def read_adj123_file(
+    gridid='psn12.5',
+    xdim=608,
+    ydim=896,
+    anc_dir='/share/apps/amsr2-cdr/nasateam2_ancillary',
```

Comment:
The `NSIDC_NFS_SHARE_DIR` constant can be utilized here (`constants.py`). Maybe we should define a new one for the `nasateam2_ancillary` dir?

Reply:
Same comment applies to `anc_dir` defaults in functions below. And a good reason to de-duplicate the string into a constant that can be more easily updated!

Reply:
This is a great idea.  Let's implement this with the extraction of the relevant code into pm_tbdata.

I've added a TODO.